### PR TITLE
update pullsecret for robot account

### PR DESCRIPTION
**Pull request by varodrig**
- Updated Devspaces to be installed using a catalog
- Update variable name for robot account when installing rhdh
- argocd roles added PipelineRun
- added a json file for the rhdh home page.

### CODE REVIEW

**Review by treddy08 — APPROVED** [review]

**Comment by varodrig** [comment]
Needs a merge. @treddy08

**Comment by varodrig** [comment]
Or anyone else if it can review/approve and merge please. @newgoliath  thanks

**Comment by varodrig** [comment]
@juliaaano

File: rhdh/deploy/templates/app-config-rhdh.yaml [hunk 1]
```diff
@@ -105,6 +105,15 @@ data:
       ocm: true
       gitlab: true
       keycloak: true
+
+    proxy:
+      '/developer-hub':
+        target: https://raw.githubusercontent.com/
+        pathRewrite:
+          '^/api/proxy/developer-hub': /varodrig/agnosticd_workload_helm_charts/main/rhdh/json/rhdh_home.json
```
Comment (juliaaano) [inline rhdh/deploy/templates/app-config-rhdh.yaml]: The path /varodrig is concerning, shouldn't it point to the upstream?

File: devspaces/values.yaml [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+nameOverride: ""
+fullnameOverride: ""
+
+namespacePermissions:
+  # Which OpenShift user gets permissions for the namespace
+  user: kubeadmin
+  # Which role to grant the user on the namespace
+  role: admin
+
+# Deploy MTA Operator
```
Comment (juliaaano) [inline devspaces/values.yaml]: It is not  really MTA here, is it?

**Review by juliaaano — COMMENTED** [review]
I am not a user or developer of this helm chart, therefore I will abstain to approve, reject or merge it.
I am not sure if merging it now can cause an impact on RH1.
Furthermore, there are too many distinct changes which makes it more difficult for me to confidently assess the impact.

**Comment by varodrig** [comment]
- Updated Devspaces to be installed using a catalog
- Update variable name for robot account when installing rhdh (approved by @treddy08 )
- argocd roles added PipelineRun
- added a json file for the rhdh home page.

**Pull request merged**